deployer/src/algolia_helper.py

```diff
@@ -44,7 +43,7 @@ def get_docsearch_key(config):
     # find a key
     for key in algolia_client_prod.list_api_keys()['keys']:
         if 'description' in key and 'docsearch frontend ' + config == key[
-            'description'] and key["acl"] == ["search"]:
+                'description'] and key["acl"] == ["search"]:
```

Comment:
Isn't python going to throw an error because of the extra tab?

Reply:
I had no issue but I'll remove it just in case